SecondProject/src/MyExample.java

```diff
@@ -7,6 +7,12 @@ public static void main(String[] args) {
 
 		System.out.println("First commit");
 		System.out.println("First commit 2 added more condition");
+		int a = 10;
+		if( a % 2 == 0) {
+			System.out.println(" even number");
+		}else {
```

Comment:
Why condition added here, instead of this you can check with simplified manner.

Reply:
Hi , my intention of doing here is very simple.